region/tourian/main.json

```diff
@@ -1418,18 +1461,33 @@
           "to": [
             {
               "id": 3,
-              "requires": null,
-              "unlock": [
-                {"or":[
-                  "Missile",
-                  "Super"
-                ]}
-              ],
-              "note": "Ammo for destroying MB1's glass?"
+              "requires": [ "Missile", "Super" ],
+              "unlock": null,
```

Comment:
This was an unlock because the glass doesn't respawn. So, like with the Zebetites, you could destroy that and go refill. I guess node 8 needs to be an event as well?

Reply:
I figure we fix all the MB stuff in a later PR anyway

Reply:
This is represented like the Zebetites though. If we make them diverge now, we may get confused later.